5e-SRD-Monsters.json

```diff
@@ -420,8 +428,13 @@
         }
       },
       {
-        "name": "Acid Breath (Recharge 5-6)",
+        "name": "Acid Breath",
         "desc": "The dragon exhales acid in a 60-foot line that is 5 feet wide. Each creature in that line must make a DC 18 Dexterity saving throw, taking 54 (12d8) acid damage on a failed save, or half as much damage on a successful one.",
+        "usage": {
```

Comment:
Thoughts on this might be that `usage` would describe the dragon using the ability, `recharge` may be better for the recharge.  Perhaps something like this:

```yaml
name: Acid Breath
Desc: The dragon exhales acid in a 60-foot line....
Usage:
   Dice: 12d8
   Average: 54
   Range:
      Type: line
      Distance: 60
      Unit: ft
Save:
   Ability: dexterity
   DC: 18
   Success: half damage 
Recharge:
   Type: roll
   Die: d6
   Success:
      Min: 4
```

Reply:
Sorry for the late reply.  And for using `yaml` in example :)

Reply:
I understand, and I would agree in the absolute, but here we have to deal with the context. We are putting the SRD in a data format. And the SRD mentions "limited usage" (see page 259). We could change "usage" to "limited_usage" if you think it's clearer.

Reply:
I have poor timing of merging things before you comment @benjaminpetersen. 😅

Reply:
clear is always good :)

And no worries, I'm not quite the most timely to respond always, tho do appreciate the effort y'all have been putting into this lately!